### ISSUE TITLE
Ban detection when unfollwing

### ISSUE THREAD

**Issue by Ardy000** [alexal1/Insomniac#248]
Please add a detection for when the actions blocked dialogue comes up during unfollow script. Currently insomniac just crashes and restarts Instagram and continues in a loop.

**Comment by alexal1** [comment]
Sounds reasonable. Could you help us by attaching such crash zip-folder? It contains UI hierarchy of such "action blocked" window

**Comment by Ardy000** [comment]
Hi Alex,
I have SOOO MANY crash logs. I will delete them all and wait for it to happen again (currently I'm banned unrelated (suspecting this is why #249))

I'm on 3.6.0 since 3.6.5 doesn't work with filters (specifically the public only privacy setting) is that an issue? any changes made in the latest release in that area?

**Comment by alexal1** [comment]
Yeah there could've been changes. Anyway, if you get a crash eventually, feel free to send logs. I'll try to reproduce on my side

**Comment by Ardy000** [comment]
Ill get you a crash log as soon as the next version after 3.6.5 is released since I can't use 3.6.5

**Comment by Ardy000** [comment]
@alexal1 I've sent you the crash logs on telegram
